docs/file_explorer.md

```diff
@@ -1,54 +1,53 @@
 
-Note that clearing the browser storage will permanently delete all the
-solidity files you wrote. To avoid this, you can use Remixd, which
-enables you to store and sync files in the browser with your local
-computer ( for more information see [remixd](remixd.html) ).
+You can rename, remove or
+add new files to the file explorer.
 
 ![](images/a-file-explorer-buttons.png)
 
-We will start by reviewing at the icons at the top left - from left to
-the right:
+We will start by reviewing the icons in the image above.
 
-Create new File
----------------
+The book icon - **A.** is the link to the module's documentation.
 
-Creates a new `untitled.sol` file in Remix.
+The icons to the right of the **browser** file explorer in the image above only appear for browser storage. 
 
-Add Local File
---------------
+Create new File
+---------------
 
-Allows you to select files from the local file system and import them to
-the Remix browser storage.
+The icon marked **B.** above. Creates a new file.
 
 Publish to Gist
 ---------------
 
-Publishes all files from the browser folder to a gist.
-Gist API has changed in 2018 and it unfortunately requires users to be authenticated to be able to publish a gist.
+The icon marked **C.** above. Publishes all files from the browser folder to a gist.  Only file in the root of **browser** will be published.  Files in subfolders will not be publish to the Gist.
+Gist API has changed in 2018 and **requires** users to be authenticated to be able to publish a gist.
+
+Click [this link](https://github.com/settings/tokens) to Github tokens setup and select Generate new token. Then check the **Create gists** checkbox and generate a new token.
 
-Click [this link](https://github.com/settings/tokens) to Github tokens setup  and select Generate new token.
-Then check only Create gists checkbox and generate a new token.
+Take the token and paste it in Remix's **Setting** module in the **Github Access Token** section. And then click Save. Now you should be able to use the feature.
+
+Create a folder
+---------------
 
-Then paste it in Remix (right panel/Settings tab) and click Save. Now you should be able to use the feature.
+The icon marked **D.** above.  Creates a new folder in **browser** file explorer.
 
-Copy to another Remix instance
-------------------------
+Context Menu (Right Clicking)
+-----------------------------
+Right click on a file or a folder and the **context menu** will appear. 
 
-Enables you to copy files from the browser storage to another instance
-(URL) of Remix.
+![](images/a-file-ex-rt-click.png)
 
-Connect your filesystem to Remix
---------------------
+You can rename or a file or a folder.  You can also delete a file or a folder. These functions also work with folders and files on your hard drive when using RemixD.  
```

Comment:
`You can rename or a file or a folder` => `You can rename a file or a folder`

Also I think, both lines can be combined as : `You can rename and delete a file or a folder`

Reply:
Updated